specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json

```diff
@@ -0,0 +1,1766 @@
+        },
+        "impactedField": {
+          "description": "The resource type identified by Advisor.",
+          "type": "string"
+        },
+        "lastUpdated": {
+          "format": "date-time",
+          "description": "The most recent time that Advisor checked the validity of the recommendation.",
+          "type": "string"
+        },
+        "shortDescription": {
+          "$ref": "#/definitions/ShortDescription",
+          "description": "A summary of the recommendation."
+        }
+      }
+    },
+    "scoreEntity": {
+      "description": "The details of Advisor Score",
+      "type": "object",
+      "properties": {
+        "date": {
+          "description": "The date score was calculated.",
+          "type": "string"
+        },
+        "score": {
+          "description": "The percentage score.",
+          "type": "number"
+        },
+        "consumptionUnits": {
+          "description": "The consumption units for the score.",
+          "type": "number"
+        },
+        "impactedResourceCount": {
+          "description": "The number of impacted resources.",
+          "type": "number"
+        },
+        "potentialScoreIncrease": {
+          "description": "The potential percentage increase in overall score at subscription level once all recommendations in this scope are implemented.",
+          "type": "number"
+        },
+        "categoryCount": {
+          "description": "The count of impacted categories.",
+          "type": "number",
+          "readOnly": true
+        }
+      }
+    },
+    "timeSeriesEntity": {
+      "description": "The historic data at different aggregation levels.",
+      "type": "array",
+      "items": {
+        "description": "The data from different aggregation levels.",
+        "type": "object",
+        "properties": {
+          "aggregationLevel": {
+            "description": "The aggregation level of the score. ",
+            "type": "string",
+            "enum": [
+              "week",
+              "day",
+              "month"
+            ],
+            "x-ms-enum": {
+              "name": "aggregated",
+              "modelAsString": true
+            }
+          },
+          "scoreHistory": {
+            "description": "The past score data",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/scoreEntity"
+            },
+            "x-ms-identifiers": []
+          }
+        }
+      },
+      "x-ms-identifiers": []
+    },
+    "advisorScoreEntity": {
+      "description": "The details of Advisor score for a single category.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The ID of the resource.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The category.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of resource.",
+          "type": "string"
+        },
```

Comment:
Is it a ProxyResource?

Reply:
@weidongxu-microsoft :  as represented here: 
<img width="422" alt="image" src="https://user-images.githubusercontent.com/105078656/216874093-bc31b254-81bf-4f18-badb-358910ac94fd.png">

https://learn.microsoft.com/en-us/rest/api/advisor/advisorscore(preview)/advisor-score/list?tabs=HTTP

Reply:
I am asking whether the schema belongs to ARM RroxyResource.
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/types.json#L91

Let ARM decide.

Reply:
@uparameswara You should consider use common type whenever is possible

Reply:
@zizw123 @weidongxu-microsoft  :  Hi, with the latest commit I pushed to generalize as proxyResource - it seems the  go and python builds broke for me 

<img width="434" alt="image" src="https://user-images.githubusercontent.com/105078656/217397902-84bb9970-1eb4-46bc-be1d-d4616dd38591.png">

Can you please review my latest commit and help to figure how i can possibly rectify the issue?

<img width="610" alt="image" src="https://user-images.githubusercontent.com/105078656/217398201-bc75934d-3433-424e-aaee-270b64688553.png">

Reply:
You JSON defined another "Resource" there.
https://github.com/Azure/azure-rest-api-specs/blob/d42498e7692c3cd2270e67e3f7bec6828c341fa1/specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json#L1337-L1357

That's a tough decision. You may use the local one this way
https://github.com/Azure/azure-rest-api-specs/blob/d42498e7692c3cd2270e67e3f7bec6828c341fa1/specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json#L1031-L1035

Again I leave this to ARM review.

Reply:
Removed the local instance of Resouce and replaced it with the common-types Resource. Thanks @weidongxu-microsoft  and @zizw123  for your prompt assistance :)